zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -146,11 +149,10 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
             MulWordByU64Gadget::construct(cb, tx_gas_price.clone(), tx_gas.expr());
         let tx_fee = cb.query_word_rlc();
 
-        // TODO: contraint l1 fee
         #[cfg(not(feature = "scroll"))]
         cb.require_equal(
-            "tx_fee == l1_fee + l2_fee, l1_fee == 0",
-            mul_gas_fee_by_gas.product().expr(),
+            "tx_fee == l1_fee + l2_fee",
```

Comment:
This condition only work when 'scroll' features is disabled (i.e. l1fee and many features just not work).

We need another constraints with `feature = "scroll"` for the `tx_fee`. Or just delete the `cfg`  annotion here, since we may be able to make a 0 l1_fee when 'scroll' features is not actived.

Reply:
Delete feature guard `scroll` for this tx-fee constraint in commit [abada55](https://github.com/scroll-tech/zkevm-circuits/pull/499/commits/abada558d37a40605762a9c3186a579d47a92d83).